contracts/ControllerRegistry.sol

```diff
@@ -9,11 +10,24 @@ contract ControllerRegistry is IControllerRegistry, Ownable {
     event ControllerRegister(address newController);
     event ControllerRemove(address newController);
 
+    /**
+     * @param _controller The address to check if registered as a controller
+     * @return Boolean representing if the address is a registered as a controller
+     */
+    function isRegistered(address _controller)
+        public
+        view
+        override
+        returns (bool)
+    {
+        return controllerRegistry[_controller];
+    }
+
     /**
      * @param _controller The address to register as a controller
      */
     function registerController(address _controller) external onlyOwner {
-        require(_controller != address(0), "Invalid address");
+        require(Address.isContract(_controller), "controller was not contract");
```

Comment:
this will fail on `address(0)` as well correct?

Reply:
Yep, unless someone manages to deploy a contract to the 0 address in which case the whole world will explode.